ui/src/app/settings/views/Configuration/Deploy/Deploy.test.tsx

```diff
@@ -16,23 +16,27 @@ describe("Deploy", () => {
     state = rootStateFactory();
   });
 
-  it("loads", () => {
+  it("loads", async () => {
     const store = mockStore(state);
 
-    const wrapper = mount(
+    render(
       <Provider store={store}>
         <Deploy />
       </Provider>
     );
-    expect(wrapper.exists()).toBe(true);
+    await waitFor(() =>
+      expect(
+        screen.getByTestId("deploy-configuration-view")
+      ).toBeInTheDocument()
+    );
   });
```

Comment:
What's your opinion on such tests (only verifying that a component has rendered)?

It seems to me that as long as the render output is covered by some other tests they don't add any real value and we could remove them.

In case such test is the only one for a particular piece of UI would be better to refactor to assert the actual output (e.g. checking if a form exists in the document).

Reply:
I think they're only useful if you're also testing the case where the component *doesn't* render, e.g with state set up this way it renders but with state set up a different way it doesn't render, or it doesn't render until the user clicks this button etc. So I don't think this test is useful at all and could definitely be removed or refactored to assert something meaningful.